### DISCUSSION TITLE
Colab for Hacktoberfest?

### DISCUSSION THREAD

**Discussion by kaustubhgupta** [avinashkranjan/Amazing-Python-Scripts#1167] [General]
Hi @avinashkranjan!

I hope you are doing well. Remember me from GSSOC? I was one of the mentors for your project. I am writing this to confirm if you would be interested in this Hacktoberfest contribution period. I was thinking if I and you accept the PRs for hacktoberfest, just like the GSSOC for your project Amazing python scripts?

Let me know your thoughts on this.
Thanks!

**Comment by avinashkranjan** [comment]
Sure @kaustubhgupta that would be a great idea 💡
